cmd/chain-maind/app/app.go

```diff
@@ -254,6 +248,7 @@ func txCommand() *cobra.Command {
 		authcmd.GetEncodeCommand(),
 		authcmd.GetDecodeCommand(),
 		flags.LineBreak,
+		vestingcli.GetTxCmd(),
```

Comment:
Do we need to add this here?

Reply:
it's possible to create vesting accounts dynamically afaik -- so this adds client side commands for it

Reply:
cc @lezzokafka With this we may be able to create vesting accounts after genesis is created.

Reply:
Just a side note: supply module calculations for liquid supply will not work for vesting accounts created after genesis.